.github/CHANGELOG.md

```diff
@@ -247,6 +247,13 @@
   connection usages, and `DeviceSpec.target` is retrieved from the device specification rather than
   passed at initialization.
 
+* The `api` subpackage has been removed and the contained `DeviceSpec` and `Result` classes have
+  been moved to the root `strawberryfields` folder. They can now be imported as follows:
+
+  ```python
+  from strawberryfields import DeviceSpec, Result
```

Comment:
Is this the preferred access pattern? Or would we rather users write
```python
import strawberryfields as sf
# sf.DeviceSpec
# sf.Result
```

Reply:
True, I think your suggestion is better.